src/backend/seeds/04_configuration.js

```diff
@@ -8,21 +8,46 @@ if (!ENV_IS_STAGING_OR_PROD) {
       .then(() =>
         // Inserts seed entries
         knex('configuration').insert([
-          { id: 0, label: 'interviews:meeting_link', value: 'www.google.com' },
+          { id: 0, label: 'interview_meeting_link', value: 'www.google.com' },
           {
             id: 1,
-            label: 'interviews:first_round_deadline',
-            value: '2022-02-22',
```

Comment:
I suggest keeping the format as 'YYYY-MM-DD' instead of 'MM-DD-YYYY' to avoid confusion.